src/pg/client.py

```
@@ -0,0 +1,312 @@
+        )
+
+        api_data = _resolve_api_data(api, client)
+
+        user_id: int | None = None
+        if fetch_user_info and client.is_connected():
+            try:
+                me = await client.get_me()
+                if me:
+                    user_id = me.id
+            except Exception:
+                pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)